src/app/products/loan-products/edit-loan-product/edit-loan-product.component.ts

```diff
@@ -175,4 +203,8 @@ export class EditLoanProductComponent implements OnInit {
       });
   }
 
+  mapStringEnumOptionToIdList(incomingValues: StringEnumOptionData[]): string[] {
```

Comment:
Use the `OptionData` already defined in 'app/shared/models/option-data.model'

Reply:
Its not the same. OptionData id is number. StringEnumOptionData id is string